platforms/corda-ent/configuration/samples/network-cordaent.yaml

```diff
@@ -93,75 +93,19 @@ network:
           ports:
             proxy: 10001
         networkmap:
-          name: networkmap
+          name: nmap
```

Comment:
why is this nmap?

Reply:
this would be used as the node name and service names, all other services refer networkmap as nmap and nmap is used in corda-ent doc as well